build.rs

```diff
@@ -14,6 +19,27 @@ fn make_protos(protos: &[&str]) {
         .unwrap();
 }
 
+fn compile_aa_interfaces() -> anyhow::Result<()> {
+    let root = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("thirdparty/account-abstraction");
+    let target_path = PathBuf::from(env::var("OUT_DIR").unwrap());
+    let build_path_config = ProjectPathsConfig::builder()
+        // current only interfaces are what we need
```

Comment:
"only interfaces are needed"

sorry for complicating so much about comments :sweat_smile:

Reply:
It is OK~! I am not a native English speaker. Don't hesitate to comment on those words as long as it could help people to understand. I can benefit myself from your comments, too.